docs/content/authorization-concepts.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Authorization Concepts
+description: Introduction to Authorization 
+sidebar_position: 1
+slug: /authorization-concepts
+---
+
+import { ProductName, ProductNameFormat, RelatedSection } from '@components/Docs';
+
+# Authorization Concepts 
+
+## Authentication vs Authorization
+
+[**Authentication**](https://auth0.com/intro-to-iam/what-is-authentication/) (or **AuthN**) ensures a user's identity. [**Authorization**](https://en.wikipedia.org/wiki/Authorization) (or **AuthZ**) determines if a user can perform a certain action on a particular resource.
```

Comment:
Is there a reason we link to Authentication docs in Auth0 but refer to Wiki for Authorization docs? https://auth0.com/docs/authenticate/protocols/oauth

Reply:
Don't bold Authentication. Only for UI text. Leaving this as a comment instead of making the change as FGA style may be different.